MonkeyLoader.Resonite.Integration/EngineInitHook.cs

```diff
@@ -167,28 +163,27 @@ private static async Task RunEngineReadyHooksAsync()
             Logger.Trace(() => "Running EngineReady hooks in this order:");
             Logger.Trace(resoniteMonkeys);
 
-            LoadProgressReporter.AdvanceFixedPhase("Executing EngineReady Hooks...");
+            LoadProgressReporter.SetSubphase("Executing EngineReady Hooks...");
             var sw = Stopwatch.StartNew();
 
             foreach (var resoniteMonkey in resoniteMonkeys)
             {
                 LoadProgressReporter.SetSubphase(resoniteMonkey.Name);
                 await LoadProgressReporter.RunForPrettySplashAsync(50, resoniteMonkey.EngineReady);
-                LoadProgressReporter.ExitSubphase();
             }
 
-            LoadProgressReporter.AdvanceFixedPhase("Determining potential Mod conflicts...");
+            LoadProgressReporter.SetSubphase("Determining potential Mod conflicts...");
             await LoadProgressReporter.RunForPrettySplashAsync(200, Mod.Loader.LogPotentialConflicts);
 
             Logger.Info(() => $"Done executing EngineReady hooks on ResoniteMonkeys in {sw.ElapsedMilliseconds}ms!");
 
-            LoadProgressReporter.AdvanceFixedPhase("Loading Fallback Locale Data for Mods...");
+            LoadProgressReporter.SetSubphase("Loading Fallback Locale Data for Mods...");
             sw.Restart();
 
             await LoadProgressReporter.RunForPrettySplashAsync(200, LocaleExtensions.LoadFallbackLocaleAsync);
             Logger.Info(() => $"Done loading fallback locale data for mods in {sw.ElapsedMilliseconds}ms!");
 
-            LoadProgressReporter.AdvanceFixedPhase("Mods Fully Loaded");
```

Comment:
No more Fixed Phase count needed? 🤔

Reply:
Okay, in the Renderer it does have `TotalFixedPhaseCount` - but it's hardcoded in the Unity Scene 🥲 
<img width="843" height="241" alt="Screenshot of the LoadProgress implementation" src="https://github.com/user-attachments/assets/7343dd9b-2552-4fa8-b00a-4d5752a58f69" />

Reply:
Okay, in the upcoming update, the load progress is going to be transmitted as a percentage, so we can properly handle it OvO